config/routes.rb

```diff
@@ -43,4 +40,20 @@
       resources :group_assignments, path: 'group-assignments'
     end
   end
+
+  namespace :stafftools do
+    mount Sidekiq::Web  => '/sidekiq'
```

Comment:
Everything under the `stafftools` namespace inherits from the stafftools controller.

This is the check: https://github.com/education/classroom/pull/359/files#diff-22dc66320c0cd133d26c6fa203093eecR4